src/sky.cpp

```diff
@@ -720,15 +720,20 @@ void Sky::update(float time_of_day, float time_brightness,
 		m_skycolor = m_mix_scolor(m_skycolor, pointcolor, m_horizon_blend() * 0.25);
 	}
 
-	float cloud_direct_brightness = 0;
+	float cloud_direct_brightness = 0.0f;
 	if (sunlight_seen) {
 		if (!m_directional_colored_fog) {
 			cloud_direct_brightness = time_brightness;
-			if (time_brightness >= 0.2 && time_brightness < 0.7)
```

Comment:
This range covered dawn and dusk periods (the periods where world brightness is changing).